app/components/sign-in.js

```diff
@@ -0,0 +1,28 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { tagName } from '@ember-decorators/component';
+import { action, computed } from '@ember/object';
+
+@tagName('')
+export default class SignInComponent extends Component {
+  @service
+  session
```

Comment:
So the `oauth2-implicit-grant` authenticator will automatically fill out/update the session once it retrieves auth params?

Reply:
Yup! It's a little bit too much "magic" but if you see the mixin in the application route, that mixin is responsible for communicating with the global "session" service.